### PR TITLE
Update to include new model with additional metadata.

### PR DESCRIPTION
**Pull request by NeilMadden**
### What

Include the latest version of the model and update to new metadata model. Also added try/catch around the dataset loading to make it more robust in the face of loading errors.

### How to review

`sbt update clean test it:test run` then load a dataset and check no errors.

### Who can review

Anyone but me.

### CODE REVIEW

File: app/actors/KafkaActor.java [hunk 1]
```diff
@@ -30,7 +31,11 @@ public KafkaActor(DataPointMapper dataPointMapper) {
                     logger.info("Received String message: {}", s);
                     ConsumerRecords<String, String> records = consumer.poll(1000);
                     List<String> jsonRecords = KafkaUtils.recordValues(records);
-                    dataPointMapper.mapDataPoints(jsonRecords);
+                    try {
+                        dataPointMapper.mapDataPoints(jsonRecords);
+                    } catch (DatapointMappingException ex) {
+                        logger.error("Caught error - discarding input: {}", ex.getMessage());
```
Comment (YetAnotherMatt) [inline app/actors/KafkaActor.java:37]: Better to log the error, including stack trace) than just the message - unless the stack trace will already have been logged or isn't useful?

File: build.sbt [hunk 2]
```diff
@@ -21,7 +21,7 @@ libraryDependencies ++= Seq(
   javaWs,
   javaJpa,
   "org.eclipse.persistence" % "eclipselink" % "2.6.2",
-  "com.github.ONSdigital" % "dp-dd-backend-model" % "develop-SNAPSHOT",
+  "com.github.ONSdigital" % "dp-dd-backend-model" % "feature~metadata-SNAPSHOT",
```
Comment (YetAnotherMatt) [inline build.sbt:24]: Should this be develop-SNAPSHOT once merged?

**Review by YetAnotherMatt — CHANGES_REQUESTED** [review]

File: app/actors/KafkaActor.java (hunk 1 above)
Comment (NeilMadden) [reply app/actors/KafkaActor.java:37]: This exception is only thrown by DataPointMapper#mapDataPoints, which already logs the full stack trace of the underlying cause before throwing this exception.

File: build.sbt (hunk 2 above)
Comment (NeilMadden) [reply build.sbt:24]: Yes, there will be a little dance to make sure nothing breaks: merge this and the API changes first, then merge the model, and finally update both projects to revert back to using develop-SNAPSHOT.

**Review by YetAnotherMatt — APPROVED** [review]

**Pull request merged**